src/modules/blockchain/blockchain-module-manager.js

```diff
@@ -529,6 +529,13 @@ class BlockchainModuleManager extends BaseModuleManager {
     async getNodeId(blockchain, identityId) {
         return this.callImplementationFunction(blockchain, 'getNodeId', [identityId]);
     }
+
+    async isKnowledgeAssetRegistered(blockchain, paranetId, knowledgeAssetId) {
```

Comment:
Do we use it anywhere?

Reply:
I needed it, but removed code using it, I thought of keeping it as it we might need it